tests/vehicle_drag_test.cpp

```diff
@@ -125,12 +137,29 @@ static bool test_drag(
     return valid;
 }
 
+/*The function print_drag_test_strings() takes a string as an argument and
+calls the function test_drag() with the corresponding vproto_id as an
+argument. Args:
+type: the type of item to be tested.
+Returns:
+Nothing.*/
```

Comment:
This doesn't need documentation, it's a surface-level description of what the function does, which is pretty clear, especially as it's directly above the quite short function. The function name is clear enough that I don't see anything here that needs to be well documented.
```suggestion
```